### ISSUE TITLE
Stack of items over 1000 items 

### ISSUE THREAD

**Issue by supergela** [P3pp3rF1y/SophisticatedBackpacks#888]
Whenever I stack more than a thousand items it, the count number is showing like this..

![Capture d’écran 2023-10-31 181614](https://github.com/P3pp3rF1y/SophisticatedBackpacks/assets/149520883/f87bf2c0-86cc-48d5-ac33-4d2f6960174a)

Game version : Forge 1.18.2

**Comment by supergela** [comment]
No one ever had this issue?

**Comment by P3pp3rF1y** [comment]
it's very likely in some way related to your locale, checking in code I don't immediately see why locale would even be considered, but will need to test this more.

**Comment by meadowsys** [comment]
I've seen iskall have this issue a long time ago while watching one of his streams (vault hunters), but I've personally not had this issue ever.

potential relevant info:
- I'm also on VH
- I'm on a Mac (macOS 14.3 developer beta at the moment, but have upgraded many times, twice across major versions, while playing VH)
- I am playing with a resource pack, but have also checked without the resource pack, and I still don't have the issue, so I don't think it is that
- I suspect its a missing font character somewhere, maybe only on windows for some reason?
- Are you using number formatting libraries, or maybe theres something you're using built into java's standard libraries (been a while since I've programmed Java so I don't know)? maybe in the backend of whatever number formatting method you're using, its formatting it differently on different systems, or different locales?
- continuing the thought of number formatting and locales, I've switched between English (UK) and Dutch (Nederlands (Nederland)) locales in minecraft a few times before, I don't remember having this issue for both. I believe my computer locale has been in Dutch the entire time I've been playing, but maybe I switched from english (UK) at some point in the middle of it so it _might_ have been an issue on english, not sure. It is respecting my locale settings and printing for example 3.000 rather than 3,000.

Random brain head thoughts, hopefully these are of some help ><
